content/en/docs/how-to/registry-authentication.md

```diff
@@ -7,7 +7,7 @@ weight: 1
 
 You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in registries that do not offer public access.
 
-> **Important** Currently, `notation` relies on Docker Credential Store for authentication, which is included in Docker Desktop. For more information, see [Docker Credential Store](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-store)
+> **Important** Currently, `notation` relies on Docker Credential Store for authentication, which is included in Docker Desktop. It requires additional configuration if you using Notation in Linux. For more information, see [Docker Credential Store](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-store).
```

Comment:
```suggestion
> **Important** Currently, `notation` relies on Docker Credential Store for authentication, which is included in Docker Desktop. It requires additional configuration if you using Notation without docker desktop, for more information, see [Docker Credential Store](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-store).
```

Reply:
Suggest re-write the 2nd sentence to make it clearer from 
~It requires additional configuration if you using Notation without docker desktop, for more information ...~
Notation requires additional configuration when used without docker desktop, for more information ...

Reply:
I find that Docker Desktop relies on a Credential Helper [pass](https://www.passwordstore.org/) to store credentials on Linux. Linux users still have some manual configuration steps as demonstrated at [Credentials management for Linux users](https://docs.docker.com/desktop/get-started/#credentials-management-for-linux-users). So we need to emphasize this configuration is used for Linux.